ui/src/underlaysSlice.ts

```diff
@@ -17,6 +16,13 @@ export type Underlay = {
   prepackagedConceptSets: PrepackagedConceptSet[];
 };
 
+export interface CriteriaConfig {
```

Comment:
Please also copy the comments and formatting.

Reply:
Done. I deleted before because I found the adjacent code all had no comments...